chapters/config.Rmd

```diff
@@ -459,6 +477,31 @@ How can we make this the default for all of our plots?
     that reads settings from a file.
```

Comment:
I'm a little confused by "an option that writes all of its settings to a file that reads settings from a file". Would you be able to write out the solution (see comment in the solutions file below)?